pages/agent/v3/debian.md

```diff
@@ -28,6 +28,9 @@ Download the Buildkite PGP key to a directory that is only writable by `root` (c
 curl -fsSL https://keys.openpgp.org/vks/v1/by-fingerprint/32A37959C2FA5C3C99EFBC32A79206696452D198 | sudo gpg --dearmor -o /usr/share/keyrings/buildkite-agent-archive-keyring.gpg
 ```
 
+>📘 Is [keys.openpgp.org](https://keys.openpgp.org) down?
+> If you get a 404 or other error from `curl` in the command above, see the [Alternative keyservers](#alternative-keyservers) section below.
```

Comment:
We generally try to avoid "above" and "below"

```suggestion
> If you get a 404 or other error from `curl` in the previous command, see the [Alternative keyservers](#alternative-keyservers) section.
```